lib/helpers/pluralize.js

```diff
@@ -6,31 +6,36 @@ module.exports = pluralize;
  * Pluralization rules.
  */
 
+// David Xie: rebase pluralization rules from pluralize.js committed on Oct 6, 2021
```

Comment:
Please remove the `// David Xie: ...` comments that mention the commits, they don't serve much value.